products/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+
+class MainCategory(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'maincategories'
+
+class SubCategory(models.Model):
+    name         = models.CharField(max_length=50)
+    maincategory = models.ForeignKey(MainCategory, on_delete=models.SET_NULL, null=True)
+
+    class Meta:
+        db_table = 'subcategories'
+
+class Product(models.Model):
+    name            = models.CharField(max_length=50)
+    creator         = models.ForeignKey('users.Creator', on_delete=models.SET_NULL, null=True, related_name='creator')
+    subcategory     = models.ForeignKey(SubCategory, on_delete=models.SET_NULL, null=True)
+    price           = models.DecimalField(max_digits=10, decimal_places=2)
+    description     = models.CharField(max_length=1000)
+    period          = models.PositiveIntegerField(null=True)
+    review_score    = models.DecimalField(max_digits=2, decimal_places=1)
+    created_at      = models.DateTimeField(auto_now_add=True)
+    updated_at      = models.DateTimeField(auto_now=True)
+    discount_rate   = models.IntegerField(null=True)
+    discount_coupon = models.IntegerField(null=True)
+
+    class Meta:
+        db_table = 'products'
+
+class ProductImage(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.SET_NULL, null=True, related_name='images')
+    url1    = models.CharField(max_length=500, null=True)
+    url2    = models.CharField(max_length=500, null=True)
+    url3    = models.CharField(max_length=500, null=True)
+    url4    = models.CharField(max_length=500, null=True)
+    url5    = models.CharField(max_length=500, null=True)
```

Comment:
왜 url이 5개나 필요한가요?

컬럼이 아니라 행(row)가 5개 생성되면 되지 않을까요?

```suggestion
class ProductImage(models.Model):
    product = models.ForeignKey(Product, on_delete=models.SET_NULL, null=True, related_name='images')
    url1    = models.CharField(max_length=500, null=True)
    url2    = models.CharField(max_length=500, null=True)
    url3    = models.CharField(max_length=500, null=True)
    url4    = models.CharField(max_length=500, null=True)
    url5    = models.CharField(max_length=500, null=True)
```